src/rust/engine/Cargo.toml

```diff
@@ -95,6 +96,7 @@ lazy_static = "1"
 log = "0.4"
 logging = { path = "logging" }
 num_enum = "0.4"
+notify = { git = "https://github.com/notify-rs/notify", rev = "fba00891d9105e2f581c69fbe415a58cb7966fdd" }
```

Comment:
Why do we need to grab this specific revision of the crate?

Reply:
There was a comment which I think got removed in a rebase or something. Ugh.

Reply:
Actually there is a comment on a later commit in this pr.